<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-HIKG7N16" failures="2" tests="9" name="Test" time="2.43" errors="0" timestamp="2024-03-29T17:16:05 ICT">
  <testcase classname="ThucHanh.GET_Invalid" name="TC01_checkStatus" time="0.007"/>
  <testcase classname="ThucHanh.GET_Invalid" name="TC02_checkMessage" time="0.003">
    <failure type="java.lang.AssertionError" message="Result havenot message expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Result havenot message expected [false] but found [true]
at ThucHanh.GET_Invalid.TC02_checkMessage(GET_Invalid.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- TC02_checkMessage -->
  <testcase classname="ThucHanh.GET_Invalid" name="TC03_MessageRes" time="0.343">
    <failure type="java.lang.AssertionError" message="message not match. expected [User not found!] but found [null]">
      <![CDATA[java.lang.AssertionError: message not match. expected [User not found!] but found [null]
at ThucHanh.GET_Invalid.TC03_MessageRes(GET_Invalid.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- TC03_MessageRes -->
  <testcase classname="ThucHanh.GET_Valid" name="TC01_checkStatus" time="0.001"/>
  <testcase classname="ThucHanh.GET_Valid" name="TC02_checkFirstName" time="0.001"/>
  <testcase classname="ThucHanh.GET_Valid" name="TC03_checkLastName" time="0.0"/>
  <testcase classname="ThucHanh.GET_Valid" name="TC04_checkAvatar" time="0.001"/>
  <testcase classname="ThucHanh.GET_Valid" name="TC05_checkUserIdMatch" time="0.027"/>
  <testcase classname="ThucHanh.GET_Valid" name="TC06_checkEmail" time="0.0"/>
</testsuite> <!-- Test -->
